meta-mbl-distro/recipes-core/images/mbl-image-production.bb

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+###############################################################################
+# mbl-image-production.bb
+#   This file is the mbed linux OpenEmbedded recipe for building a minimal
+#   uboot/kernel/filesystem image only including production packages.
+###############################################################################
+inherit mbl-distro-image-production
+
+SUMMARY = "Mbed Linux Basic Minimal Image With Production Packages"
+DESCRIPTION = "Image with production packages."
```

Comment:
It's strange that the description is shorter and less descriptive than the summary... Can we just use the same text for both?

Reply:
Adressed in 816f1c0.